### ISSUE TITLE
no subject no email no link

### ISSUE THREAD

**Issue by Djinnie** [thundernest/import-export-tools-ng#404]
Hello,
I used to save my emails by exporting a folder in html and I hadn't any problem until, I'm not sure, last update of Thunderbird or maybe the installation of Google-4-TBsync.
Now when I try to export a folder, it seems to work but the html file doesn't show any subject (unless it's a reply or forward or if there's no subject) and so no link. No mention of any email (from or to), the date used to be in French format, it's now in US format. The name of the files used to be date+subject (even if it's not easy for French people because of accented letters), it's now email+date (but I think, it's better).
In short, it seems there's something wrong with the fields and I don't know what to do to fix that...

I'm on Ubuntu 20.04.6 and TB 102.10.0, both in French

Any idea?
Thank you in advance

**Comment by cleidigh** [comment]
@Djinnie 
Hmm, that seems like a bunch of things scrambled!
Let's start with the simple things : 
- Disable all other extensions and retry
- Export a single message that you could post here and see what that outputs 
- Capture a screenshot of the filenames setup tab

Accents are not an issue so I am not clear what you are referring to there.
@cleidigh

**Comment by Djinnie** [comment]
Thank you @cleidigh 
I already tried to disable all, even uninstall Google_4_TBsync, uninstall et reinstall ImportExport, tried thunderbird in English, tried others options of ImportExport (without attachment, in text), the option in one file seems to work (but because it's just one file, I'm not sure the app calls any of the fields).

For a single test message, this is what I get
`<html>
<head>
<style>
table { border-collapse: collapse; }
th { background-color: #e6ffff; }
th, td { padding: 4px; text-align: left; vertical-align: center; }
tr:nth-child(even) { background-color: #f0f0f0; }
tr:nth-child(odd) { background-color: #fff; }
tr>:nth-child(5) { text-align: center; }
</style>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>test</title>
</head>
<body>
<h2>test (01/05/2023 21:28:34)</h2><table width="99%" border="1" ><tr><th><b>Sujet </b></th><th><b>De </b></th><th><b>Pour </b></th><th><b>Date </b></th><th><b>Pièce(s) jointe(s) </b></th></tr>
<tr><td><a href="messages/Virginie%20Levenok-20230501-1.html"></a></td>
<td></td>
<td>(none)</td>
<td nowrap>5/01/2023 21.27</td>
<td align="center">&nbsp;</td></tr></table></body></html>`

I didn't know for the setup!  (but it will fix my accented letters problem) 

Thank you
![setup](https://user-images.githubusercontent.com/132290109/235521075-f55ecf3c-c4be-4f38-b54d-f9eba250690e.png)

**Comment by cleidigh** [comment]
@Djinnie 
Ok first thing is that the example output you gave is a one line index file. What menu and menu option did you use?
For a single message export you would right click on the message and choose : 
Format HTML

Unless you also want an index, then choose the With HTML index/HTML index
@cleidigh

**Comment by Djinnie** [comment]
Well, I wasn't clear! The messages are ok, only the index file is wrong. 
I save my messages by folder with the option html with attachment

**Comment by cleidigh** [comment]
@Djinnie 
ok I tested the message export with index (in French locale) and the index was good: Are you doing this also, try the message context menu.

**Comment by cleidigh** [comment]
@Djinnie 
Here is my example with a small folder : 

```html
<html>
<head>
<style>
table { border-collapse: collapse; }
th { background-color: #e6ffff; }
th, td { padding: 4px; text-align: left; vertical-align: center; }
tr:nth-child(even) { background-color: #f0f0f0; }
tr:nth-child(odd) { background-color: #fff; }
tr>:nth-child(5) { text-align: center; }
</style>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>2014</title>
</head>
<body>
<h2>2014 (5/01/2023)</h2><table width="99%" border="1" ><tr><th><b>Sujet </b></th><th><b>De </b></th><th><b>Pour </b></th><th><b>Date </b></th><th><b>Pièce(s) jointe(s) </b></th></tr>
<tr><td><a href="messages/Uh-oh...%20your%20transfer%20is%20about%20to%20expire%20and%20it%20will%20be%20deleted%20soon.-%24%7Bdestinat%C3%A1rio%7D-%24%7Bassunto%7D.html">Uh-oh... your transfer is about to expire and it will be deleted soon.</a></td>
<td>WeTransfer &lt;noreply@wetransfer.com&gt;</td>
<td>test1@kokkini.net" &lt;test1@kokkini.net&gt;</td>
<td nowrap>10/12/2014</td>
<td align="center">&nbsp;</td></tr>
<tr><td><a href="messages/from-%24%7Bdestinat%C3%A1rio%7D-%24%7Bassunto%7D.html">from</a></td>
<td>Christopher Leidigh &lt;test1@kokkini.net&gt;</td>
<td>test1@kokkini.net</td>
<td nowrap>7/05/2022</td>
<td align="center">&nbsp;</td></tr>
<tr><td><a href="messages/af%20from-%24%7Bdestinat%C3%A1rio%7D-%24%7Bassunto%7D.html">af from</a></td>
<td>Christopher Leidigh &lt;test1@kokkini.net&gt;</td>
<td>test1@kokkini.net</td>
<td nowrap>7/05/2022</td>
<td align="center">&nbsp;</td></tr></table></body></html>
```

**Comment by cleidigh** [comment]
@Djinnie 
Our posts crossed and I can't get gh to show my html rendered as opposed to the code?

Is your last example using the message context menu?
@cleidigh

**Comment by cleidigh** [comment]
@Djinnie 
So the above is from the message context menu? Does this happen with any message you choose?
@cleidigh

**Comment by Djinnie** [comment]
I tried different messages of course. Two things are very weird for me, it's that if the message is a reply, you get the RE and the link! And that you get a "none" for the email adress...
<html>
<head>
<style>
table { border-collapse: collapse; }
th { background-color: #e6ffff; }
th, td { padding: 4px; text-align: left; vertical-align: center; }
tr:nth-child(even) { background-color: #f0f0f0; }
tr:nth-child(odd) { background-color: #fff; }
tr>:nth-child(5) { text-align: center; }
</style>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>test</title>
</head>
<body>
<h2>test (01/05/2023 23:23:39)</h2><table width="99%" border="1" ><tr><th><b>Sujet </b></th><th><b>De </b></th><th><b>Pour </b></th><th><b>Date </b></th><th><b>Pièce(s) jointe(s) </b></th></tr>
<tr><td><a href="20230501-Re_testons-Virginie_Levenok.html">Re: </a></td>
<td></td>
<td>(none)</td>
<td nowrap>5/01/2023 23.09</td>
<td align="center">&nbsp;</td></tr></table></body></html>

**Comment by cleidigh** [comment]
@Djinnie 
I agree it is very odd. I am on Windows so now I think I have to test with my Linux VM to eliminate platform issues. That will take me a bit and I have to go offline in just a few minutes so will be tomorrow before I can report back.
The other test you can try is export to EML with index.
If you could post a simple email perhaps with just a slight change to make the private email address changed then we could use the same thing for testing.
@cleidigh

**Comment by Djinnie** [comment]
I'm not sure to understand. What's the difference if I try EML as only the index file doesn't work and it's always the same one?
And for the records it was working on the very same ubuntu until maybe my last update of TB

**Comment by cleidigh** [comment]
@Djinnie 
I am just suggesting different approaches as I am perplexed. I managed to get my Linux VM (Mint 23) with French locale and updated to TB 102.10 up. I was able to do one test. I exported from the message context menu with index and the message and index were as expected.

After the TB 102 update have you noticed anything else not running correctly?
Would you mind sending me an email then I can send you back a test email? I  am thinking that I may need to get you a version with debugging output because I see no problem here including Linux.
My email : 
test1@kokkini.net
@cleidigh

**Comment by Djinnie** [comment]
@cleidigh 
No everything seems running as expected.
Maybe I can uninstall and reinstall TB... I just need to find a way to save my setup (I can't bother configuring again)

**Comment by cleidigh** [comment]
@Djinnie 
I would try creating a new test profile, I am wondering if there was some issue updating.
@cleidigh

**Comment by cleidigh** [comment]
@Djinnie 
Any luck or update?
@cleidigh

**Comment by Djinnie** [comment]
@cleidigh 
No, but my thunderbird is up to date and I didn't try to uninstall it (I got some deadlines so it will wait better times)

**Comment by cleidigh** [comment]
@Djinnie 
I am going to close, but we can reopen whenever you wish.
@cleidigh
